charts/datarepo/Chart.yaml

```diff
@@ -46,9 +46,7 @@ dependencies:
   version: 0.0.6
   repository: https://broadinstitute.github.io/datarepo-helm/
   condition: create-secret-manager-secret.enabled
-  # uncomment after cloud deployment of just de-elasticsearch completes,
-  # and updates the chart.lock and index.yaml
-#- name: de-elasticsearch
-#  version: 0.1.0
-#  repository: https://broadinstitute.github.io/datarepo-helm/
-#  condition: de-elasticsearch.enabled
```

Comment:
I didn’t mention it before, but I wonder if this needs to be `de-elasticsearch`. We don’t plan to have uses of ES other than data explorer. So maybe `elasticsearch` would be sufficient. Or is it hard to change it at this point?

Reply:
I wanted to name it de-elasticsearch to distinguish it from the official es helm chart, which is just called "elasticsearch"

Reply:
I think this is a useful distinction to have for debugging.